src/test/store/publish.test.js

```diff
@@ -281,7 +281,9 @@ describe('attemptToPublish', function() {
     expect(await publishAttempt).toEqual(true);
 
     expect(getUploadPhase(getState())).toEqual('uploaded');
-    expect(getUploadGeneration(getState())).toEqual(1);
+    // The generation is incremented twice because of some asynchronous code in
+    // the uploader function.
+    expect(getUploadGeneration(getState())).toEqual(2);
```

Comment:
I wonder if this should be a greater than check, to avoid testing implementation details.

Reply:
good idea!